### DISCUSSION TITLE
Help: possibility of relative filepaths i.e. /Users/sandersantema -> /home/sandersantema

### DISCUSSION THREAD

**Discussion by sandersantema** [beetbox/beets#3935] [Q&A] [answered]
I was wondering whether there is any way to use relative instead of absolute paths, this is the last step in my journey to setting up a pipeline from beets on linux to beets on my mac to iTunes (almost there!). 

I use the same folder structure on both machines but `$HOME` differs, i.e. on linux it's `/home/sandersantema/Music` while it's `/Users/sandersantema/Music` on macOS. Any solutions to this which wouldn't require an ugly symlink or fiddling around with manually changing path's in the SQL database?

**Comment by jackwilsdon** [comment, marked as answer]
I think all of our media paths are absolute at the moment, take a look at #133.

**Reply by sandersantema** [reply]
That's the issue I was searching for, thanks! Manually performing 
```
sqlite3 ~/.config/beets/library.db "UPDATE items SET path = replace(path, '/Users/sandersantema', '/home/sandersantema');"
```

Although I've now unfortunately run into a different issue which I've opened an issue for.
